dbprocessing/dbprocessing.py

```diff
@@ -259,7 +258,9 @@ def figureProduct(self, filename=None):
         claimed = []
         for code, desc, arg, product in act_insp:
             try:
-                inspect = imp.load_source('inspect', code)
+                fname = code
```

Comment:
Is this assignment necessary? Can't we just pass "code" directly to load_source?